DigitalOffice/Services/ProjectService/Commands/DisableWorkersInProjectCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Commands
+{
+    public class DisableWorkersInProjectCommand : IDisableWorkersInProjectCommand
+    {
+        private readonly IProjectRepository repository;
+        private readonly IValidator<WorkersIdsInProjectRequest> validator;
+
+        public DisableWorkersInProjectCommand(
+            [FromServices] IProjectRepository repository,
+            [FromServices] IValidator<WorkersIdsInProjectRequest> validator)
+        {
+            this.repository = repository;
+            this.validator = validator;
+        }
+
+        public bool Execute(WorkersIdsInProjectRequest request)
+        {
+            if (request == null)
```

Comment:
no need, we have validator for such thing

Reply:
Fixed